larsim/LegacyLArG4/OpFastScintillation.hh

```diff
@@ -259,6 +259,13 @@ namespace larg4 {
     void getVISTimes(std::vector<double>& arrivalTimes, const TVector3 &ScintPoint, const TVector3 &OpDetPoint);
     // Visible component timing parameterisation
 
+    void detectedDirectHits(std::map<size_t, int>& DetectedNum,
+                            const double Num,
+                            geo::Point_t const& ScintPoint);
+    void detectedReflecHits(std::map<size_t, int>& ReflDetectedNum,
+                            const double Num,
+                            geo::Point_t const& ScintPoint);
```

Comment:
Now that these functions are public, is it possible to mark them as `const`?  This may require marking other member functions as `const` also.  The goal is to avoid adjusting the internal state of the class from outside.

Reply:
Yes, good point, thanks. Done with commit 8721ec9.